app/components/ui/Modal/Modal.jsx

```diff
@@ -1,60 +0,0 @@
-import React, { Fragment } from 'react';
-import PropTypes from 'prop-types';
-import { noop } from 'lodash';
-import Modal from 'react-modal';
-
-import styles from './Modal.scss';
-
-export const BaseModal = props => {
```

Comment:
Any reason to keep this around for the modal work we were planning on doing on the edit pages?

Reply:
Actually I take that back, I don't think this is worth hanging on to. It's quick enough to implement when we need it, and a lot of this was specifically geared towards this specific implementation that took over the full page. Would rather delete it all now so we don't end up with junk we don't need later.

Reply:
Yeah, and this will always be available in the Git history for us to take a look at in case if we want to reference it in the future.